### DISCUSSION TITLE
Defi or Dex ?

### DISCUSSION THREAD

**Discussion by siriusjack19** [IBAX-io/go-ibax#161] [Ideas]
I think defi is much better,becouse of an open financial system run with smart contracts currently used by the Ethereum cryptocurrency platform, one of the best cryptocurrencies after bitcoin. The existence of this smart contract can make it easy for crypto money users to carry out transactions and investments without having to be intermediaries.

**Comment by hrhrrcp** [comment]
Bu aqdum maillerini nasıl iptal edicem sürekli geliyor

iPhone’umdan gönderildi

siriusjack19 ***@***.******@***.***>> şunları yazdı (23 Oca 2022 16:37):


I think defi is much better,becouse of an open financial system run with smart contracts currently used by the Ethereum cryptocurrency platform, one of the best cryptocurrencies after bitcoin. The existence of this smart contract can make it easy for crypto money users to carry out transactions and investments without having to be intermediaries.

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/161>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXL3UYYZJQU2RRSQD6Y7AF3UXQADHANCNFSM5MTLN2AA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
